### PR TITLE
Add changelog for v1.0.0

### PR DESCRIPTION
**Pull request by mpoke**
# Description

Changelog for v1.0.0, aka Replicated Security. 

## Linked issues

Closes: https://github.com/cosmos/interchain-security/issues/670

## Type of change

- [x] `Docs`: Adds documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+In addition, RS has the following features:
+
+- **Key Assignment**: Enables validator operators to use different consensus keys for each consumer chain validator node that they operate.
+- **Jail Throttling**: Enables the provider to slow down a "worst case scenario" attack where a malicious consumer binary attempts to jail a significant amount (> 2/3) of the voting power, effectively taking control of the provider.
+
+### Dependencies
+
+- [ibc-go](https://github.com/cosmos/ibc-go): [v4.2.0](https://github.com/cosmos/ibc-go/blob/release/v4.2.x/CHANGELOG.md)
+- [cosmos-sdk](https://github.com/cosmos/cosmos-sdk): [v0.45.11-ics](https://github.com/cosmos/cosmos-sdk/tree/v0.45.11-ics)
```
Comment (mpoke) [inline CHANGELOG.md:32]: @sainoe please bump to v0.45.12-ics once a release is cut.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# CHANGELOG
+
+## v1.0.0
+
+Date: TBA
```
Comment (mpoke) [inline CHANGELOG.md:5]: TODO: please update

Comment (jtremback) [inline CHANGELOG.md:5]:
```suggestion
Date: TBA
```
Date: February 6th, 2023

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- **Validator Set Update**: Enables the provider to 
+  (1) update the consumers on the voting power granted to validators (based on the changes in the active validator set on the provider chain), 
+  and (2) ensure the timely completion of unbonding operations (e.g., undelegations).
+- **Consumer Initiated Slashing**: Enables the provider to jail validators for downtime infractions on the consumer chains. 
+- **Reward Distribution**: Enables the consumers to transfer to the provider (over IBC) a portion of their block rewards as payment for the security provided. Once transferred, these rewards are distributed on the provider using the protocol in the [distribution module of Cosmos SDK](https://docs.cosmos.network/v0.45/modules/distribution/). 
+- **Consumer Chain Removal**: Enables the provider to remove a consumer either after a `ConsumerRemovalProposal` passes governance or after one of the timeout periods elapses -- `InitTimeoutPeriod`, `VscTimeoutPeriod`, `IBCTimeoutPeriod`.
+
```
Comment (jtremback) [inline CHANGELOG.md:23]:
```suggestion

```
- **Social Slashing**: Equivocation offenses (double signing etc.) on consumer chains are logged, and then can be used in a governance proposal to slash the validators responsible.

**Pull request merged**